build_tools/benchmarks/export_benchmark_config.py

```diff
@@ -131,12 +122,99 @@ def main(args: argparse.Namespace):
         "run_configs": serialization.serialize_and_pack(run_configs),
     }
 
-  json_data = json.dumps(output_map, indent=2)
+  return output_map
+
+
+def _compile_stats_handler(_args: argparse.Namespace):
+  all_gen_configs, _ = benchmark_collections.generate_benchmarks()
+  compile_stats_gen_configs = [
+      config for config in all_gen_configs
+      if benchmark_collections.COMPILE_STATS_TAG in config.compile_config.tags
+  ]
+
+  return serialization.serialize_and_pack(compile_stats_gen_configs)
+
+
+def _parse_and_strip_list_argument(arg) -> List[str]:
+  return [part.strip() for part in arg.split(",")]
+
+
+def _parse_benchmark_presets(arg) -> List[PresetMatcher]:
+  matchers = []
+  for preset in _parse_and_strip_list_argument(arg):
+    matcher = BENCHMARK_PRESET_MATCHERS.get(preset)
+    if matcher is None:
+      raise argparse.ArgumentTypeError(
+          f"Unrecognized benchmark preset: '{preset}'.")
+    matchers.append(matcher)
+  return matchers
+
+
+def _parse_arguments():
+  """Parses command-line options."""
+
+  # Makes global options come *after* command.
+  # See https://stackoverflow.com/q/23296695
+  subparser_base = argparse.ArgumentParser(add_help=False)
+  subparser_base.add_argument("--output",
+                              type=pathlib.Path,
+                              help="Path to write the JSON output.")
+
+  parser = argparse.ArgumentParser(
+      formatter_class=argparse.RawDescriptionHelpFormatter,
+      description=textwrap.dedent("""
+      Export type: "benchmark" outputs a list of object:
```

Comment:
I don't think "benchmark" is a good way to categorize the distinction here. This whole script is for exporting benchmark configurations and compilation statistics are also benchmarks. A benchmark just involves measuring something in some standard way for comparison. I think the distinction here is that these are "module execution benchmarks". "stats" also seems a bit redundant here. All of these are also statistics. So I think I would call these two categories "execution" and "compilation"

Reply:
SG. Done.